packages/react/src/components/Tab/Tab.tsx

```diff
@@ -115,41 +182,39 @@ export default class Tab extends React.Component {
       handleTabClick,
       handleTabKeyDown,
       disabled,
+      // @ts-expect-error: Deprecated prop
```

Comment:
we want to include these in the component props interface even if they're deprecated